CHANGELOG.md

```diff
@@ -6,8 +6,20 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## NEXT - TBD
 
+### Added
+
+### Fixed
+
+## [0.3.1] - 2021-03-09
+### Added
+- FSDP docs ([#455](https://github.com/facebookresearch/fairscale/issues/455))
+- enable\_wrap and auto\_wrap APIs ([#446](https://github.com/facebookresearch/fairscale/issues/446))
+- experimental offload module ([#432](https://github.com/facebookresearch/fairscale/issues/432))
+
 ### Fixed
-- ShardedDDP auto catch trailing buckets (TBD)
+- Many FSDP fixes
+- Several SDP fixes around performance and corner cases
```

Comment:
feel free to add more details here by doing GitHub "suggesting". Then, it is able to keep the co-author info.